selenium_setup.sh

```diff
@@ -0,0 +1,17 @@
+# Selenium headless browser
```

Comment:
The current convention is that we've separated the install into OS-level (apt), and by language. I'm open to the idea of rather doing it on a logical package basis (e.g. `selenium_setup.sh`, `ipyleaflet_setup.sh`, etc.), but I would prefer we stick to one convention.

What do you think?